src/cljs/athens/devcards/right_sidebar.cljs

```diff
@@ -1,13 +1,13 @@
 (ns athens.devcards.right-sidebar
   (:require
-    [athens.views.buttons :refer [button]]
+    ["/components/Button/Button" :refer [Button]]
     [athens.views.right-sidebar :as right-sidebar]
     [devcards.core :refer [defcard-rg]]
     [re-frame.core :refer [dispatch]]))
 
 
 (defcard-rg Toggle
-  [button {:primary true :on-click-fn #(dispatch [:right-sidebar/toggle])} "Toggle"])
+  [:> Button {:isPrimary true :on-click-fn #(dispatch [:right-sidebar/toggle])} "Toggle"])
```

Comment:
Reagent will convert props in dash-chase into camelCase. So we can still keep using idiomatic property names in cljs.

Reply:
Instead of using `:>` everywhere, we can also declare these components with `reagent/adapt-react-class` somewhere. Not sure if it's worth the effort though.

Reply:
I prefer dash-case.

I prefer `:>`.